src/Twitter.php

```diff
@@ -1446,16 +1497,17 @@ public function init(string $path, Http\Client $client) : void
      * returned verbatim. Otherwise, a zero is returned.
      *
      * @param string|int $int
-     * @return string|int
      */
-    protected function validInteger($int)
+    protected function validInteger($int) : int
     {
-        if (is_int($int)) {
+        if (is_int($int) && $int > -1) {
```

Comment:
I realized in reviewing tests that negative values aren't allowed, so I added this additional condition.